src/applications/gi/sass/partials/_gi-vet-tec.scss

```diff
@@ -19,4 +19,12 @@
     margin: 0;
   }
 
+  @media (min-width: 641px) {
```

Comment:
Why is this breakpoint chosen?  The [design system](https://design.va.gov/design/breakpoints) already has some specified.

Reply:
That is the width in which we are swapping from single column view.

Reply:
Ok.  The design system provides a way for you to [create a custom breakpoint](https://design.va.gov/design/breakpoints#what-if-you-need-to-create-a-different-breakpoint) like `$single-column-width`.  Something like this might work:

```scss
$single-column-width: 641px;

// ...

@include media($single-column-width) {
    .single-column-display-block{
      display: block !important; }
  }

```

Reply:
Okay, I've added in the change.